lib/imports/api/authority-registry/server/methods.js

```diff
@@ -131,11 +132,26 @@ export const removeDocFromAuthorityServer = new ValidatedMethod({
     });
   },
   run ({ docId, fullUrl, groupId = null }) {
+    let result = {
```

Comment:
this `removeDocFromAuthority` method was only partway implemented, I completed the implementation so that if you remove a doc from an Authority and that doc was the last of its group (via groupId), that groupId is also removed from the Authority server in the registry